### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by Penguronik**
Created new structure for existing tracking antenna code and changed some drivers

### CODE REVIEW

File: TrackingAntenna/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        "cmath": "cpp"
+    }
+}
```
Comment (HardyYu) [inline TrackingAntenna/.vscode/settings.json]: no need to upload this setting file

File: TrackingAntenna/Src/Util/Util.cpp [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+  // int rawDifference = bearing - AAT_BEARING; // this is how he was doing it, figure out what he's really doing here and whats necessary
+  int rawDifference = bearing;
+
+  if (rawDifference > 180) {
+      rawDifference = rawDifference - 360;
+  } else if (rawDifference < -180) {
+      rawDifference = rawDifference + 360;
+  }
+  if (rawDifference > 85) {
```
Comment (HardyYu) [inline TrackingAntenna/Src/Util/Util.cpp]: The 85 and -85 here are magic numbers. Could be probably create some constant to hold them with a name.

File: TrackingAntenna/Src/AntennaDynamics/AntennaDynamics.cpp [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    PDEBUG("\n");
+}
+
+// Function to set the yaw angle of the tracker in degrees
+void AntennaDynamics::setPitchAngle(float angle) {
+    if (angle > 90 || angle < 0) {
+        return;
+    }
+    int p_microseconds;
+    p_microseconds = map(angle, 0, 90, 575, 2300);
```
Comment (HardyYu) [inline TrackingAntenna/Src/AntennaDynamics/AntennaDynamics.cpp]: what are these numbers? Could we add some comments to elaborate that or create some constants to hold them?

**Review by HardyYu — COMMENTED** [review]
Good work! I couldn't find any big issue with the current cod e structure. Definitely 100 times  more organized than before. Added some minor change suggestions for easier maintenance in the future.
An idea of a feature we probably could add in the future, having a TAProfile class which gives different options for this tracking antenna to run. For example, when this profile of TA doesn't work, we can quickly switch to another profile which allows the antenna tower operator to manual control the tracker with keyboard. All they change is the function they call in the profile.

File: TrackingAntenna/Src/Util/Util.cpp (hunk 2 above)
Comment (eekaba) [reply TrackingAntenna/Src/Util/Util.cpp]: 85 and -85 refer to the maximum rotation of a standard servo in each direction (~180 degrees). These can be changed based on the specific servo's limits.

**Review by HardyYu — APPROVED** [review]
👍

**Pull request merged**